### PR TITLE
Run code formatter on tokenizer_text.exs

### PR DESCRIPTION
**Pull request by nbw**
Formatter worked great. 

## Note
These sorts of expressions feel a bit awkward, but I can't really think of a better way and it's just a test:

screenshot 2017-10-11 11 08 01

### CODE REVIEW

File: lib/eex/test/eex/tokenizer_test.exs [hunk 1]
```diff
@@ -13,148 +13,176 @@ defmodule EEx.TokenizerTest do
...
+    assert T.tokenize(string, 1) == {
+             :ok,
+             [
+               {:text, 'foo '},
+               {:expr, 1, '=', ' bar\n\nbaz '},
+               {:text, '\n'},
+               {:expr, 4, '', ' foo '},
+               {:text, '\n'}
+             ]
+           }
```
Comment (josevalim) [inline lib/eex/test/eex/tokenizer_test.exs]: You can rewrite those to this:

```elixir
    exprs = [
      {:text, 'foo '},
      {:expr, 1, '=', ' bar\n\nbaz '},
      {:text, '\n'},
      {:expr, 4, '', ' foo '},
      {:text, '\n'}
    ]

    assert T.tokenize(string, 1) == {:ok, exprs}
```

And then please make sure to run the formatter again.

File: lib/eex/test/eex/tokenizer_test.exs [hunk 2]
```diff
@@ -13,148 +13,176 @@ defmodule EEx.TokenizerTest do
...
+               {:text, ' e %> f'}
+             ]
+           }
+
+    assert T.tokenize('<%%% a <%%= b %> c %>', 1) == {
+             :ok,
+             [
+               {:text, '<%% a <%= b %> c %>'}
+             ]
+           }
```
Comment (josevalim) [inline lib/eex/test/eex/tokenizer_test.exs]: This one (and the ones that return a single expression only) you can probably rewrite it to this:

```elixir
 assert T.tokenize('<%%% a <%%= b %> c %>', 1) ==
         {:ok, [{:text, '<%% a <%= b %> c %>'}]}
```

Don't forget to format again after all your changes.

File: lib/eex/test/eex/tokenizer_test.exs [hunk 3]
```diff
@@ -13,148 +13,173 @@ defmodule EEx.TokenizerTest do
...
       {:expr, 1, '=', ' d '},
       {:text, ' e %> f'}
-    ]}
+    ]
+
+    assert T.tokenize('a <%% b <%= c %> <%= d %> e %> f', 1) == {:ok, exprs}
+  end
 
-    assert T.tokenize('<%%% a <%%= b %> c %>', 1) == {:ok, [
+  test "improperly formatted quotation with interpolation" do
```
Comment (nbw) [inline lib/eex/test/eex/tokenizer_test.exs]: This test originally belongs to "quotation with interpolation" on line 65 (there were originally two assertions in the test). I've moved it out and labeled it "improperly formatted quotation", which is what it is as far as I can tell.

**Comment by nbw** [comment]
1. moved args into `exprs` variables as suggested
2. split one of the tests into two tests (comment above)
3. reran formatter

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**